docs/rest-framework.rst

```diff
@@ -19,3 +19,29 @@ You can create a Mutation based on a serializer by using the
         class Meta:
             serializer_class = MySerializer
 
+
+Customizing the mutation
+--------
+
+By default, when the serializer is instantiated, only the input argument
+is passed. Sometimes, the Django `request` object is required in the serializer
+context. In fact, any sort of complicated serializer will probably require something
+like `request.user`. This can be performed by customizing the instantiation
+method as such:
+
+.. code:: python
+
+    from graphene_django.rest_framework.mutation import SerializerMutation
+
+    class MySecondAwesomeMutation(SerializerMutation):
+        
+        @classmethod
+        def instantiate_serializer(cls, instance, args, request, info):
```

Comment:
I just had a quick look at drf source, it's indeed `get_serializer` everywhere. I'll make the change.